src/ItemEdit.js

```diff
@@ -139,9 +139,17 @@ const ItemEdit = ({ index, roadmap, onChange, onDone }) => {
 
   const handleSuggestionSelect = (index, event) => {
     const values = [...dateFields];
-    if (event.target.name && event.target.name.includes('Stage')) {
+    if (
+      event.target.innerText &&
+      roadmap.labels.map((label) => label.name).includes(event.target.innerText)
```

Comment:
Why do we need this test? Can't we just check if the name is "Stage" then we set `.stage` to whatever the suggestion is?

Reply:
Same as above, there is no `event.target.name` on the event anymore. Maybe there is more digging to be done as to why that no longer exists on the event.

Reply:
I think we want a way to reliably map the key we are setting to the field being changed.

I wonder if the `name` isn't available anymore for reasons similar to the recent Select change you made to ensure that clicking a suggestion fires an `onChange`?

Reply:
Yes that seems correct. The change event that comes from typing something into the input has an `event.target.name` however the event coming from a TextInput with suggestions selection does not because it's a click event.

Reply:
I've done some digging, this functionality with `event.target.name` worked properly until Grommet `2.16.3`, it no longer works. The reason being is that previously, when users clicked on a TextInput suggestion, the `target` of the click event was registered as the underlying `input`, which does have `name` attribute.
<img width="364" alt="Screen Shot 2021-04-16 at 11 23 06 AM" src="https://user-images.githubusercontent.com/12522275/115068581-49cf9500-9ea7-11eb-887a-d994d69dbe4e.png">

However, starting in Grommet `2.16.3`, the target of that onClick event when clicking on a suggestion became the suggestion's div which does not have the `name` attribute.
<img width="403" alt="Screen Shot 2021-04-16 at 11 27 41 AM" src="https://user-images.githubusercontent.com/12522275/115068669-68ce2700-9ea7-11eb-86a3-4450f928719d.png">

I think the question is, is this change in event.target desired/correct? Or is this a bug in grommet?

Reply:
This feels like a bug in grommet. I think selecting a suggestion needs to trigger a `change` event from the underlying `<input />` DOM element in order for uncontrolled Forms to work. It would be good to verify this.